control-plane/subcommand/gateway-resources/command.go

```diff
@@ -159,8 +174,14 @@ func (c *Command) Run(args []string) int {
 		return 1
 	}
 
-	// Load config from the configmap.
-	if err := c.loadConfig(); err != nil {
+	// Load apigw resource config from the configmap.
+	if err := c.loadAPIGWResourceConfig(); err != nil {
+		c.UI.Error(fmt.Sprintf("Error loading config: %s", err))
```

Comment:
~ Might be a good idea to differentiate config error message `c.UI.Error(fmt.Sprintf("Error loading api gw config: %s", err))`